ocaml/sdk-gen/common/CommonFunctions.mli

```diff
@@ -0,0 +1,130 @@
+(** Exception for unknown wire protocol. *)
+exception Unknown_wire_protocol
+
+(** Type representing supported protocols. *)
+type wireProtocol = XmlRpc | JsonRpc
+
+val get_release_branding : string -> string
+(** [get_release_branding codename] Gets the branding for a release codename.
+    @param codename Release codename to lookup.
+    @return Branding for the release codename, or the original codename if not found. *)
+
+val is_setter : Datamodel_types.message -> bool
+(** [is_setter message] Checks if a message is a setter based on its name.
+    @param message Message to check.
+    @return [true] if the message is a setter, [false] otherwise. *)
+
+val finally : (unit -> 'a) -> always:(unit -> unit) -> 'a
```

Comment:
This should be coming from the `Fun.protect` and not be required (or just an alias)

Reply:
I've replaced all usages of `finally` with `Fun.protect`, instead